prometheus_client/utils.py

```diff
@@ -22,3 +24,11 @@ def floatToGoString(d):
             mantissa = f'{s[0]}.{s[1:dot]}{s[dot + 1:]}'.rstrip('0.')
             return f'{mantissa}e+0{dot - 1}'
         return s
+
+
+
+def getMultiprocDir() -> str:
```

Comment:
```suggestion
def _getMultiprocDir() -> str:
```
Just want to make sure this is explicitly private to avoid anyone depending on it...